draft-ietf-tls-rfc8446bis.md

```diff
@@ -4217,8 +4217,7 @@ missing_extension:
 
 unsupported_extension:
 : Sent by endpoints receiving any handshake message containing an extension
-  known to be prohibited for inclusion in the given handshake message, or including
-  any extensions in a ServerHello or Certificate not first offered in the
+  in a ServerHello or Certificate not first offered in the
```

Comment:
```suggestion
  in a ServerHello, Certificate, or EncryptedExtension not first offered in the
```
@tomato42 Like so?

Reply:
or...

```suggestion
  in a ServerHello, HelloRetryRequest, EncryptedExtensions, or Certificate not first offered in the
```

Reply:
@martinthomson yes, I think we need `ServerHello, HelloRetryRequest, EncryptedExtensions, or Certificate` to actually be exhaustive